### ISSUE TITLE
FATA[0000] Not a go project

### ISSUE THREAD

**Issue by Dr-Terrible** [omeid/kar#1]
Please, can you document how to correctly install `Kargar`, `Gonzo`, `Kar`, and whatever is required to properly use the command-line tool?

At the moment, when I invoke `Kar` I only got this short error message:

```
FATA[0000] Not a go project. 
```

**Comment by omeid** [comment]
Sure thing, I have been a little busy with work but I will have a complete working demo project by end of this weekend. :)

**Comment by Dr-Terrible** [comment]
Thank you @omeid.
I'm looking forward to new commits.

**Comment by Dr-Terrible** [comment]
Hi @omeid. Is there any progress on the issue?

**Comment by omeid** [comment]
Sorry for the delay. I will promise to have something for you in less than a day. Work has been hectic busy.

**Comment by omeid** [comment]
Sorry man, busy days. But I still managed to put something together, checkout this [Angular Material Golang](https://github.com/omeid/golang-angular-material) example that is using Kar, Kargar, and Gonzo, and a brand new BoltDB wrapper and such!

let me know if you run into any problems.

**Comment by Dr-Terrible** [comment]
Thank you, @omeid.
I'm going to read it and hack with {Kar,Kargar,Gonzo} during the weekend. I'll report back to you any useful feedback or bugs I'll found.

**Comment by Dr-Terrible** [comment]
Just to let you know, so far everything is running smoothly without any sort of issues.

The error that I have reported initially ( FATA[0000] Not a go project. ) was the consequence of an orphaned `./bin/kar` executable that was compiled from 529ce46 but never removed from the main $PATH (messing up my $GOPATH somehow); After I have removed the orphaned binary and updated-compiled kar from 337e4a1, the issue disappeared.

Now I'm able to correctly run kar in my own projects and from your [Angular Material Golang](https://github.com/omeid/golang-angular-material) example too.

Thank you very much for the time you spent writing that example, I really appreciate it.
